website/documentation/appendix/releasenotes/0.10.0-M1.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: documentation
+title: Release Notes
+---
+{% include base.html %}
+## Release Notes - 0.10.0-M1
+
+We are proud to announce the release of 0.10.0-M1.  
```

Comment:
Do not mention release! Instead write sth like: "We are proud to announce the milestone 0.10.0-M1 of Eclipse Vorto.